apps/tutorial_servo_control_can/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include "mbed.h"
+#include "TutorialServo/TutorialServo.h"
+
+CAN canReceive(p30, p29);
+CANMessage msg;
+float percent
+
+int main(){
+    TutorialServo servo(PA_1);
+    CANMessage msgRec;
+     if(can.read(msgRec)) {
+        rxMsg >> percent;
```

Comment:
rxMsg is not declared in this scope. it might be helpful to think of the overloaded operator >> as (some_obj.member_function)